suite-native/graph/package.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "@suite-native/graph",
```

Comment:
I am not sure about name, I would probably suggest something more specific like `home-graph` because there is already reducer so it can be only used in one place, because it will share state everywhere.

Reply:
Is a separate package for the graph is really needed? 🤔

Reply:
It's kind of standalone functionality with lot of logic/components around that has potential to grow even more. 

If I place this to the `module-home` package for example, it can work too, but it not that future proof. Over the time `module-home` will grow and could change quite often. With every change you need to rerun lint, type-check, tests for whole `module-home`. With graph as package, you are reducing amount of these checks needed to run, it's partially explained here https://www.notion.so/satoshilabs/Code-improvements-9-breaking-the-monolith-528bf80c34ba4a6bba3f335e60ff29e8#402adaee3192443d8ce477a970207d63

There is also not-a-low potential that we will need to use graph in more that one place and it's much easier to just little bit tweak this package to be reusable then cut it out-of much bigger package.